src/messages/forget/publish.ts

```diff
@@ -38,42 +40,44 @@ type ForgetPublishConfiguration = {
  *
  * @param configuration The configuration used to publish the forget message.
  */
-export async function Publish(configuration: ForgetPublishConfiguration): Promise<ForgetMessage> {
+export async function Publish({
+    account,
+    APIServer = DEFAULT_API_V2,
+    hashes,
+    reason,
+    channel,
+    storageEngine = ItemType.inline,
+    inlineRequested,
+}: ForgetPublishConfiguration): Promise<ForgetMessage> {
+    if (inlineRequested) console.warn("inlineRequested is deprecated and will be removed: use storageEngine.inline");
+
     const timestamp = Date.now() / 1000;
-    const content: ForgetContent = {
-        address: configuration.account.address,
+    const forgetContent: ForgetContent = {
+        address: account.address,
         time: timestamp,
-        hashes: configuration.hashes,
-        reason: configuration.reason ? configuration.reason : "None",
+        hashes: hashes,
+        reason: reason || "None",
```

Comment:
Why "None" as a string ?

Reply:
Not sure I remember the original reason for this behavior.
Should this field be `undefined` if there is no reason ?

Reply:
This should fix this: https://github.com/aleph-im/aleph-sdk-ts/pull/110/commits/9e4f52e5e05b4c9208b013f5bba8143d81183b4c

Reply:
I guess this is due to the fact that "None" is the Python null type.

Reply:
This field should remain null if it's null. "None" ends up stored as a string.